src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1912,6 +1912,7 @@ private function registerValidationConfiguration(array $config, ContainerBuilder
         if (!class_exists(ExpressionLanguage::class)) {
             $container->removeDefinition('validator.expression_language');
             $container->removeDefinition('validator.expression_language_provider');
+            $container->removeDefinition('controller.expression_language');
```

Comment:
I don't think this belongs inside `registerValidationConfiguration()`, since it has nothing to do with the Validator component. It should be moved to the `load()` method, as that's where `web.php` is loaded.